examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -485,6 +519,10 @@ angular.module('cloudberry.common', ['cloudberry.mapresultcache'])
           case "totalCount":
             cloudberryService.commonTotalCount = result.value[0][0].count;
             break;
+          case "pinDBtweet":
+            cloudberryConfig.tweetDBresult = result.value[0][0];
+            $rootScope.$emit("TweetData",cloudberryConfig.tweetDBresult);
```

Comment:
This one... Why we need to use $rootScope?

Reply:
This way of communicating between pointmap.js and service.js is tried in my first PR by Qiushi, teyu and me. So I talk with Qiushi and decide to use this method to finish the communication between the two javascript files.